documentation/2-options.md

```diff
@@ -918,6 +918,28 @@ By default, requests will not use [method rewriting](https://datatracker.ietf.or
 
 For example, when sending a `POST` request and receiving a `302`, it will resend the body to the new location using the same HTTP method (`POST` in this case). To rewrite the request as `GET`, set this option to `true`.
 
+### `enableUnixSocket`
+**Type: `boolean`**\
+**Default: `false`**
+
+Set to `true` to enable unix socket domain.
+
```

Comment:
There should be a note to about recommending not to enable this option if you accept untrusted user input for the URL.